### DISCUSSION TITLE
xfce working line color

### DISCUSSION THREAD

**Discussion by arky217** [SpiralLinux/SpiralLinux-project#488] [Q&A]
No matter what I set the text color to in the xfce terminal,
the bottom line that indicates the percentage of completion
is always orange, which is nearly impossible to read on
a black background.
Is there ANY WAY to change it to white or green or
any color but orange ?

**Comment by geckolinux** [comment]
Hmm do you have a screenshot? Maybe it's coming from the GTK theme? I'm really not sure.
